src/lib/Auth.svelte

```diff
@@ -92,38 +64,6 @@
       </div>
     </form>
 
-    <p class="description">Don't have an account? Sign up here.</p>
-    <form class="form-widget" on:submit|preventDefault={handleSignUp}>
-      <div>
-        <label for="email">Signup Email</label>
-        <input
-          id="email"
-          class="inputField"
-          type="email"
-          placeholder="Your email"
-          bind:value={signUpEmail}
-        />
-      </div>
-      <div>
-        <label for="password">Create a Password</label>
-        <input
-          id="password"
-          class="inputField"
-          type="password"
-          placeholder="Your password"
-          bind:value={signUpPassword}
-        />
-      </div>
-      <div>
-        <button
-          type="submit"
-          class="button block"
-          aria-live="polite"
-          disabled={loadingSignIn}
-        >
-          <span>{loadingSignUp ? 'Loading' : 'Complete Signup'}</span>
-        </button>
-      </div>
-    </form>
+    <p class="description">Don't have an account? Sign up <a href="/signup">here</a>.</p>
```

Comment:
I'm actually getting a 404 when I click on this link. I scanned the documentation, and it looks like this might be fixed by either replacing the anchor tag with `<Link>` for by adding the `use:link` attribute to the existing anchor tag: https://github.com/EmilTholin/svelte-routing#link-1

This would need to be done on the Signup page as well for the link back to Login, probably

Reply:
That is odd, as I checked it again and was not getting a 404 on my local machine. But I saw on the version that was deployed that it was giving a 404. 

Anyway, I added the `use:link` attribute to the anchor tags. Give it a try and let me know if it works. Here's proof that it works on my machine:

[screen-recorder-sat-oct-15-2022-16-10-20.webm](https://user-images.githubusercontent.com/19156328/196005879-c5c9e3bc-9987-4c82-a6c4-3c085e685412.webm)

Reply:
@bprussell huh, that's really strange. I wonder why it was working in the dev environment but not in the deployment.

At any rate, this is working. Thanks for your contribution!